docs_src/microservices/general/index.md

```diff
@@ -202,6 +205,40 @@ Thus, if your device name is 温度感测器 (a Simplified Chinese character str
 curl http://localhost:59881/api/{{ api_version }}/device/name/温度感测器
 ```
 
+### Enable the NameFieldEscape
+EdgeX services communicate with each other by HTTP API or MQTT broker, and the communicate might fail when using the HTTP or MQTT reserved characters. 
+
+- The URL reserved characters: `! * ' ( ) ; : @ & = + $ , / ? # [ ]`
+- The MQTT reserved characters: `# +`
+
+For example, the device name is `TestDevice/Virtual`, we should encode the `/` to `%2F` in curl command. 
```

Comment:
provide the mapping table for these encoding

Reply:
I removed this and rewrote the table and example.